packages/devtools_app/macos/Flutter/GeneratedPluginRegistrant.swift

```diff
@@ -5,8 +5,10 @@
 import FlutterMacOS
 import Foundation
 
+import path_provider_macos
```

Comment:
revert this file as this seems unrelated. We should add GeneratedPluginRegistrant.swift to the `.gitignore` file and remove it from github.